subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -1326,8 +1339,81 @@ uint32_t radio_tmr_start(uint8_t trx, uint32_t ticks_start, uint32_t remainder)
 
 	nrf_timer_cc_set(EVENT_TIMER, 0, remainder);
 
+#if defined(CONFIG_BT_CTLR_NRF_GRTC)
+	uint32_t l, h, ho, stale;
```

Comment:
Are one-letter variables common this these type of files?

What about `l` => `cntr_l`? 
Why is the counter for `H` called both `h` and `ho`?

Reply:
updated